hf_vision_model_onnx_gke/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8
+
+WORKDIR /app
+
+# install dependencies
+COPY requirements.txt /app/requirements.txt
+RUN pip install --no-cache-dir --upgrade -r /app/requirements.txt
+
+# copy fastAPI app codebase
+COPY main.py /app/main.py
+COPY utils.py /app/utils.py
+
+# copy the onnx model file
+COPY vit-base-patch16-224.onnx /app/vit-base-patch16-224.onnx
```

Comment:
`.onnx` file is missing. maybe it would be better to release the model if the `.onnx` is produced by `convert-to-onnx.ipynb`?

Reply:
Done.